tests/data/wire/ext_conf_multiple_extensions.xml

```diff
@@ -16,31 +16,11 @@
-                location="https://mock-goal-state/zrdfepirv2sn3prdstr06.blob.core.windows.net/3055d769a0ed485d8c5ea8a7cd24073e/Microsoft.Powershell_ExampleExtension_ussouth_manifest.xml"
-                state="enabled" autoUpgrade="true"
-                failoverlocation="https://mock-goal-state/zrdfepirv2sn4prdstr02a.blob.core.windows.net/3055d769a0ed485d8c5ea8a7cd24073e/Microsoft.Powershell_ExampleExtension_ussouth_manifest.xml"
-                runAsStartupTask="false" isJson="true" useExactVersion="false"/>
-        <Plugin name="Microsoft.EnterpriseCloud.Monitoring.ExampleHandlerLinux" version="1.0.0"
-                location="https://mock-goal-state/rdfepirv2sn3prdstr01.blob.core.windows.net/8b23413e0ce64aa89889a295b6122112/Microsoft.EnterpriseCloud.Monitoring_ExampleHandlerLinux_ussouth_manifest.xml"
-                state="enabled" autoUpgrade="true"
-                failoverlocation="https://mock-goal-state/ardfepirv2sn3prdstr05.blob.core.windows.net/8b23413e0ce64aa89889a295b6122112/Microsoft.EnterpriseCloud.Monitoring_ExampleHandlerLinux_ussouth_manifest.xml"
-                runAsStartupTask="false" isJson="true" useExactVersion="false"/>
-        <Plugin name="Microsoft.CPlat.Core.ExampleExtensionLinux" version="1.0.0"
-                location="https://mock-goal-state/zrdfepirv2sn4prdstr09a.blob.core.windows.net/f72653efd9e349ed9842c8b99e4c1712/Microsoft.CPlat.Core_ExampleExtensionLinux_ussouth_manifest.xml"
-                state="enabled" autoUpgrade="true"
-                failoverlocation="https://mock-goal-state/ardfepirv2sn3prdstr05.blob.core.windows.net/f72653efd9e349ed9842c8b99e4c1712/Microsoft.CPlat.Core_ExampleExtensionLinux_ussouth_manifest.xml"
-                runAsStartupTask="false" isJson="true" useExactVersion="false"/>
-        <Plugin name="Microsoft.OSTCExtensions.Edp.ExampleExtensionLinuxInTest" version="1.0.0"
-                location="https://mock-goal-state/zrdfepirv2sn6prdstr05a.blob.core.windows.net/b01058962be54ceca550a390fa5ff064/Microsoft.OSTCExtensions.Edp_ExampleExtensionLinuxInTest_ussouth_manifest.xml"
-                state="enabled" autoUpgrade="true"
-                failoverlocation="https://mock-goal-state/rdfepirv2sn3prdstr01.blob.core.windows.net/b01058962be54ceca550a390fa5ff064/Microsoft.OSTCExtensions.Edp_ExampleExtensionLinuxInTest_ussouth_manifest.xml"
-                runAsStartupTask="false" isJson="true" useExactVersion="false"/>
+        <Plugin name="OSTCExtensions.ExampleHandlerLinux" version="1.0.0" location="http://mock-goal-state/rdfepirv2hknprdstr03.blob.core.windows.net/b01058962be54ceca550a390fa5ff064/Microsoft.OSTCExtensions_ExampleHandlerLinux_asiaeast_manifest.xml" config="" state="enabled" autoUpgrade="false" failoverlocation="http://mock-goal-state/rdfepirv2hknprdstr04.blob.core.windows.net/b01058962be54ceca550a390fa5ff064/Microsoft.OSTCExtensions_ExampleHandlerLinux_asiaeast_manifest.xml" runAsStartupTask="false" isJson="true"/>
```

Comment:
Had to reformat this file as it was breaking the regex for the `mockwiredata.set_extensions_config_state` function.

Reply:
It was breaking an existing regex or it started breaking due to your changes? Can you expand on the changes in this file?

Reply:
It was breaking an existing regex. I think we never tested the regex on this file before so that's why we didnt know. I mainly just reformatted it and removed the extra spaces and new line chars and made it 1 coherent line (per Plugin line).